src/lib/index.ts

```diff
@@ -56,6 +59,7 @@ class ServerlessIamPerFunctionPlugin {
           properties: {
             iamRoleStatementsInherit: { type: 'boolean' },
             iamRoleStatementsName: { type: 'string' },
+            iamPermissionsBoundary: { type: 'string' },
```

Comment:
Should be clarified and replaced with [`'#/definitions/awsArnString'`](https://github.com/serverless/serverless/blob/v2.16.1/lib/plugins/aws/provider.js#L257)

Reply:
updated.